gef.py

```diff
@@ -6416,7 +6416,7 @@ def do_invoke(self, argv: List[str]) -> None:
             if "all" in argv:
                 tids = [t.num for t in threads]
             else:
-                tids = self.check_thread_ids(argv)
+                tids = self.check_thread_ids([int(a) for a in argv])
```

Comment:
this was a list of strings before, which didn't follow the type hint for `check_thread_ids`.